modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -47,64 +53,29 @@ Click *View* to see the name, type, and expiration of the secret group.
-
-If many, or all, of the formats are selected, it can take about 70 minutes to build the table for the service, as a table with all formats is approximately 2 GB in size.
-
-image::tokenization-example-create-tokenization-service.png[]
-
-Once you create the tokenization service, you need to create an API Gateway to route requests to the tokenization service.
-
-== Create an API Gateway for the Tokenization Service
-
-Go to the *Tokenization Service* page in Runtime Manager to get the information you need for the implementation URL.
-
-. In Runtime Manager, click *Tokenization Service* in the menu on the left.
+Tokenization involves sensitive data, and therefore, works only with TLS. You must download the automatically generated API gateway Mule app and manually add TLS to the API gateway Mule app connectors before the tokenization service will work. +
+Once you create the tokenization service, you need to create an API gateway to route requests to the tokenization service.
+. In API Manager, xref:2.x@api-manager::getting-started-proxy.adoc[set up the API gateway] for the Mule app to serve as the proxy. 
+.. Go to the *Tokenization Service* page in Runtime Manager to get the information you need for the implementation URL.
+.. In Runtime Manager, click *Tokenization Service* in the menu on the left.
 . Click *Edit* for the tokenization service for which to create the API Gateway.
 +
 image::tokenization-example-edit-token-service.png[]
-. Confirm the Runtime Fabric assignment. +
-The tokenization service name is “mytoken1” and the implementation URL will be: “https://mytoken1­tokenizer:3443”.
-
-The tokenization service listens on port 3443 and is available through HTTPS only. The hostname portion is formed by taking the “Service Name” + “tokenizer” to arrive at the Kubernetes service name. The service name in this example is “mytoken1”, so the hostname is "mytoken1tokenizer".
-
-image::tokenization-example-create-tokenization-service.png[]
-
+.. Confirm the Runtime Fabric assignment. +
++
+The tokenization service listens on port 3443 and is available through HTTPS only. The hostname portion is formed by taking the “Service Name” + “tokenizer” to arrive at the service name. For example, if the tokenization service name is “mytoken1” the implementation URL will be: “https://mytoken1­tokenizer:3443”. 
+ 
 == Create an API from the Tokenization RAML
 
-Once you have the information you need to set up a routable tokenization service using an Api Gateway, create an API from the tokenization RAML.
+Once you have the information you need to set up a routable tokenization service using an API Gateway, create an API from the tokenization RAML.
```

Comment:
Wait, we have never said routable before? Are there instructions for a non-routable toke. service? Do we need "routable"?